pyiceberg/catalog/__init__.py

```diff
@@ -675,8 +675,11 @@ def _convert_schema_if_needed(schema: Union[Schema, "pa.Schema"]) -> Schema:
 
             from pyiceberg.io.pyarrow import _ConvertToIcebergWithoutIDs, visit_pyarrow
 
+            downcast_ns_timestamp_to_us = Config().get_bool("downcast-ns-timestamp-to-us-on-write") or False
```

Comment:
Nit: we can move `"downcast-ns-timestamp-to-us-on-write"` into a constant, and reuse it in `pyarrow.py`

Reply:
Thank you for the review! I've adopted this in the new commits